src/modelfree/reward_shaping.py

```diff
@@ -0,0 +1,241 @@
+from abc import ABC, abstractmethod
+import numpy as np
+import logging
+import time
+from baselines.common.vec_env import VecEnvWrapper
+from baselines.bench.monitor import ResultsWriter
+from modelfree.simulation_utils import ResettableAgent
+from gym.envs.mujoco import HumanoidEnv
+
+
+class RewardShapingEnv(VecEnvWrapper):
```

Comment:
We've already discussed verbally some changes to this, deferring review until after it's been updated.